modin/pandas/general.py

```diff
@@ -759,3 +759,11 @@ def get_names(obj):
         return list(names[0]) if is_list_like(names[0]) else [names[0]]
     else:
         return None
+
+
+def to_timedelta(arg, unit=None, errors="raise"):
+    if isinstance(arg, Series):
+        arg = arg._to_pandas()
+    result = pandas.to_timedelta(arg, unit=unit, errors=errors)
+    if isinstance(result, pandas.Series):
+        return Series(result)
```

Comment:
What would happen if `result` is _not_ a `pandas.Series`?.. this function would return `None` then, which is weird.
It feels like it should be an assert instead, something like
```suggestion
    assert isinstance(result, pandas.Series), f"Expected pandas.Series, got {type(result)}"
    return Series(result)
```

Reply:
I believe the output type is dependent on the input type. From pandas documentation:
```
If parsing succeeded. Return type depends on input:
list-like: TimedeltaIndex of timedelta64 dtype
Series: Series of timedelta64 dtype
scalar: Timedelta
```